meeting-note/meeting-note.qml

```diff
@@ -32,6 +33,13 @@ QtObject {
             "type": "string",
             "default": "meeting",
         },
+        {
+            "identifier": "timeInNoteName",
+            "name": "Time in note name",
+            "description": "Add time (HH:mm) in 'Meeting' note name.",
+            "type": "boolean",
+            "default": true,
```

Comment:
I think it would be best to default to false since it's a new feature that otherwise will modify the behavior of the current installations

Reply:
great, I'm waiting for the commit ;)